tests/console/login.go

```diff
@@ -36,21 +41,23 @@ func LoginToCirros(vmi *v1.VirtualMachineInstance) error {
 	if err != nil {
 		return err
 	}
-	_, _, err = expecter.Expect(regexp.MustCompile(`\$`), 5*time.Second)
+	_, _, err = expecter.Expect(regexp.MustCompile(`\$`), promptTimeout)
 	if err == nil {
 		return nil
 	}
 
-	b := append([]expect.Batcher{
+	b := []expect.Batcher{
 		&expect.BSnd{S: "\n"},
 		&expect.BExp{R: "login as 'cirros' user. default password: 'gocubsgo'. use 'sudo' for root."},
 		&expect.BSnd{S: "\n"},
 		&expect.BExp{R: hostName + " login:"},
 		&expect.BSnd{S: "cirros\n"},
 		&expect.BExp{R: "Password:"},
 		&expect.BSnd{S: "gocubsgo\n"},
-		&expect.BExp{R: PromptExpression}})
-	resp, err := expecter.ExpectBatch(b, 180*time.Second)
+		&expect.BExp{R: PromptExpression},
+	}
+	const loginTimeout = 180 * time.Second
```

Comment:
nit: `loginTimeout` is used in several place. Does it make sense to make it a global const?

Reply:
There is one occurrence where it uses 2sec instead of 180sec. So there is no on-value-fit-all.